### DISCUSSION TITLE
在今年的周年庆中，我想自动标记BOSS主怪这个指令能不能加上

### DISCUSSION THREAD

**Discussion by Boluo-XD** [lisai9093/YYS#21] [Q&A]
因为很多情况下，对于萌新或者小号，如果不标记BOSS的话阿修罗就会打偏，导致速度成倍，想咨询下这个功能可以加吗，感谢作者支持作者！

**Comment by lisai9093** [comment]
目前还没有加入标记功能的打算因为有一定的难度。你可以尝试加入boss截图看看，然后直接用pull request添加到这里。

**Reply by Boluo-XD** [reply]
感谢作者及时的回复！今天号被查到了，劝诫大家要收敛一点。好尴尬...

**Reply by lisai9093** [reply]
建议不要长时间挂同一个副本超过2小时以上。。。脚本有限制御魂副本次数所以一般没问题，其他副本还是尽量小心

**Reply by Boluo-XD** [reply]
再次对作者表示感谢！我这几天周年庆一直在挂999，是我的问题，作者的脚本还是很优秀的！

**Reply by chiello0** [reply]
> 感谢作者及时的回复！今天号被查到了，劝诫大家要收敛一点。好尴尬...

哈哈   我也被抓了   有点尴尬   还是不能用太久

**Reply by potato156514** [reply]
> 建议不要长时间挂同一个副本超过2小时以上。。。脚本有限制御魂副本次数所以一般没问题，其他副本还是尽量小心

个人有个不成熟的想法不知道有没有用，该脚本应该是因为长时间的机械式运作而容易被检测，若添加在战斗过程中随机移动鼠标或随机点击无用区域甚至添加AI，打开聊天界面在附近或者世界频道进行简单的聊天或者应声虫操作是否会好一些，或者在操作鼠标进行点击时添加一定的随机延时和对鼠标点击的位置的随机offset。让程序由机械化走向随机化，是否会减少被查到脚本的概率

**Reply by lisai9093** [reply]
> > 建议不要长时间挂同一个副本超过2小时以上。。。脚本有限制御魂副本次数所以一般没问题，其他副本还是尽量小心
> 
> 个人有个不成熟的想法不知道有没有用，该脚本应该是因为长时间的机械式运作而容易被检测，若添加在战斗过程中随机移动鼠标或随机点击无用区域甚至添加AI，打开聊天界面在附近或者世界频道进行简单的聊天或者应声虫操作是否会好一些，或者在操作鼠标进行点击时添加一定的随机延时和对鼠标点击的位置的随机offset。让程序由机械化走向随机化，是否会减少被查到脚本的概率

脚本已经内置随机点击和随机等待。不过我设置的比较激进因为我想更快刷完副本。目前同一个副本两小时内基本没啥问题，我也就没有继续优化了。
